en/docs/get-started/configure-users-and-roles.md

```diff
@@ -66,4 +66,79 @@ Now you have started the servers, let’s create the users and define their perm
     ![set_new_user_roles](../assets/img/get-started/quick-start-guide/set-new-user-roles.png)
         
     iii. Click **Finish**.
+
+## Configuring SMS OTP Authenticator
```

Comment:
we'll have to move this section to a different page.

Reply:
Add this section as a new page under
Get Started > QSG > below the  "Configure Users and Roles" page > Add new page

Duplicate the same content under
Install and setup > below the  "Configure Users and Roles" page > Add new page

Reply:
Add SMS OTP to a new page